store/cart.ts

```diff
@@ -1,118 +1,34 @@
-import { Module } from 'vuex'
-import { CartState } from '../types/Cart'
```

Comment:
Why remove types?

Reply:
First it was removed because it couldn't be used on a empty state, but then I remove the state, so maybe I can set it to `setCart(state, payload) { state.response = payload }` now(?)

But at the same time I don't think we can have a full specification (interface?) for the api result because we don't know exactly which values in the response is being sent for each specific case. We can still specify the values that we will use, but other values being sent from the API might change.

Should we change the types in `types/cart.ts` to only include properties we will use?

Reply:
> Should we change the types in types/cart.ts to only include properties we will use?

I think this is the best solution!

Reply:
And we only use this;
```
export interface CartState {
  entries: Array<{
    product: CartProduct
  }>
}
```

Reply:
nah, this looks better, right?
```
export interface CartState {
  entries: {
    product: {
      code: String
    }
  }
}
```